src/zorak/cogs/admin/admin_block_exe.py

```diff
@@ -0,0 +1,62 @@
+"""
+Listener to scan for exe or zip files.
+"""
+import logging
+
+from discord.ext import commands
+
+from zorak.utilities.cog_helpers._embeds import embed_cant_do_that
+
+logger = logging.getLogger(__name__)
+
+
+class FiletypeChecker(commands.Cog):
+    """
+    Listener to check if a exe file or a zip file is attached in
+    a user's message to prevent malicious files from being sent.
+    """
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.Cog.listener()
+    async def on_message(self, message):
+        for attachment in message.attachments:
+            print(f"content type: {attachment.content_type}")
+            if attachment.content_type in [
```

Comment:
Nice. Love the idea of blocking lots of different file types, As most of these are not really necessary.